crates/cheatcodes/src/inspector.rs

```diff
@@ -2172,12 +2115,12 @@ impl Cheatcodes {
 
                     op::MLOAD => {
                         // The offset of the mload operation is at the top of the stack
-                        let offset = try_or_return!(interpreter.stack().peek(0)).saturating_to::<u64>();
+                        let offset = try_or_return!(interpreter.stack.peek(0)).saturating_to::<u64>();
 
                         // If the offset being loaded is >= than the memory size, the
                         // memory is being expanded. If none of the allowed ranges contain
                         // [offset, offset + 32), memory has been unexpectedly mutated.
-                        if offset >= interpreter.shared_memory.len() as u64 && !ranges.iter().any(|range| {
+                        if offset >= interpreter.memory.size() as u64 && !ranges.iter().any(|range| {
```

Comment:
I think this is equivalent, double check

Reply:
Confirmed: https://github.com/bluealloy/revm/blob/202bb36f45ae8512143fe2d56348aa023406ce81/crates/interpreter/src/interpreter/shared_memory.rs#L55-L57